git_resource/test/Tiltfile

```diff
@@ -1,6 +1,34 @@
 os.putenv('TILT_GIT_RESOURCE_CHECKOUT_DIR', os.path.abspath('./.git-sources'))
 load('../Tiltfile', 'git_checkout', 'deploy_from_dir')
 
+symbols = load_dynamic('../Tiltfile')
+
+url_test_cases = [
```

Comment:
I'm sure it matches Tilt's coding standard, but the forced-line-wrapping here makes this super difficult to read

Reply:
Food for thought:
```python
url_test_cases = [
  ('https://github.com/tilt-dev/tilt-example-html',        ('https://github.com/tilt-dev/tilt-example-html', 'tilt-example-html', ''                 )),
  ('git@example.com/path/to/repo',                         ('git@example.com/path/to/repo',                  'repo',              ''                 )),
  ('git@example.com/path/to/repo.git',                     ('git@example.com/path/to/repo.git',              'repo',              ''                 )),
  ('git@example.com/path/to/repo.git@myRevisionSha',       ('git@example.com/path/to/repo.git',              'repo',              '@myRevisionSha'   )),
  ('git@example.com/path/to/repo.git#myBranchNameHere',    ('git@example.com/path/to/repo.git',              'repo',              '#myBranchNameHere')),
  ('https://user@example.com/path/repo.git',               ('https://user@example.com/path/repo.git',        'repo',              ''                 )),
  ('ssh://user@example.com:54321/path/repo.git',           ('ssh://user@example.com:54321/path/repo.git',    'repo',              ''                 )),
  ('https://user@example.com/path/repo.git@myRevisionSha', ('https://user@example.com/path/repo.git',        'repo',              '@myRevisionSha'   )),
]
```

Reply:
ya, totally agree, i'm going to create a helper function to make this more readable